test-integration/helpers/ui/user.coffee

```diff
@@ -122,7 +129,8 @@ class User extends TestHelper
     # Going to the root URL while logged in will redirect to dashboard
     # which may redirect to the course page.
     @test.utils.verbose('Waiting until tutor-js page loads up')
-    @test.driver.wait(selenium.until.elementLocated(css: '#react-root-container .-hamburger-menu'))
```

Comment:
ah, i removed this because i pulled this out the selector, assigned it to `el.menu` and put the get [here](https://github.com/openstax/tutor-js/pull/1020/files#diff-d29979bbcae4d70277ab0f7af8fd7bd5R133), which will wait until the `#react-root-container .-hamburger-menu` is displayed before continuing